saleor/settings.py

```diff
@@ -647,3 +647,14 @@ def get_host():
 
 assert hasattr(schema_printer, "_print_object")
 schema_printer._print_object = patched_print_object
+
+# Settings for populatedb command
```

Comment:
Do we really need to have these envs for `populateDB` script inside a `settings.py` file?  It is something that is a useful only for local development or sandboxes but I am not sure that we should place it here. Maybe we can put them directly in the command file? And use there `os.environ` also?
@maarcingebala  What do you think? settings.py is growing and growing not sure that we should place here settings which are required only by a command line.

Reply:
I agree, now if I think on it, I would consider going even further and drop the env var (use hardcoded constant). It feels unlikely to me that someone will ever gonna make a use of it, the command arg seems more appealing.